test/validation/org/apache/cassandra/bridges/ctoolbridge/CToolBridge.java

```diff
@@ -137,9 +140,9 @@ public String readClusterLogs(String testName)
         String combinedResult = "";
         String existingFolder = CASSANDRA_DIR + "/build/test/logs/validation/ctoolbridge/" + testName;
 
-        if(ArchiveClusterLogs.checkForFolder(existingFolder))
+        if (ArchiveClusterLogs.checkForFolder(existingFolder))
         {
-            for(int count = 1; count <= nodeCount; count++)
+            for (int count = 1; count <= nodeCount; count++)
```

Comment:
No there is no technical need to this. I'll make the changes, will take hardly a minute.